### ISSUE TITLE
CTS subcases should detect duplicates

### ISSUE THREAD

**Issue by zoddicus** [gpuweb/cts#2401]
Currently the unit tests will catch when we duplicate test cases, but the actual CTS runners do not catch duplicates in the cases being run.

If possible we should detect duplicate test cases being run for things like the numeric testing, since a lot of this code is programmatically generating cases.

**Comment by kainino0x** [comment]
The CTS presubmit should fail if there are duplicate cases. (It used to be that the page would fail to load at all but I think we moved the check elsewhere to speed up page load.) I don't remember, does that check not apply to subcases?

**Comment by zoddicus** [comment]
I don't think it applies to subcases, I have never had failures in the fp testing code for duplicate subcases, and I am pretty sure I don't write perfect code :upside_down_face:

I filed this as a bug to remind myself to take a look into this, when I have some spare cycles.
